deploy/olm-catalog/mobile-developer-console-operator/0.2.6/mdc_v1alpha1_mobiledeveloperconsole_crd.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This looks to be the MobileClient crd but the file name indicates it's the mobiledeveloperconsole crd. I think we need both?

Reply:
this file was generated by the operator-sdk but I also included in the manifest repo the mobile client crd https://github.com/integr8ly/manifests/tree/master/integreatly-mobile-developer-console/0.2.6

Reply:
I think what @laurafitzgerald is saying is that if this is the file that the operator-sdk generated, then something has gone wrong: the name of the file doesn't correspond to the CRD inside (and we at least need an extra one for the one mentioned in the name).

Reply:
Yes, possible something went wrong as I needed to copy the mobile client crd from the `deploy` folder and add to the mdc manifest folder.